src/pkg/projects/Microsoft.NETCore.App/src/Microsoft.NETCore.App.depproj

```diff
@@ -34,6 +34,9 @@
     <PackageReference Include="Microsoft.NETCore.Platforms">
       <Version>$(MicrosoftNETCorePlatformsPackageVersion)</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.NETCore.Targets">
+      <Version>2.0.0</Version>
```

Comment:
I think you should define a property for this to follow the existing pattern.  That way we can change it with dependency flow.  And also, we can have maestro update this version just like the other package references above.

Reply:
We don't intend to change it.  We produced one last version that was 2.0.0 with an empty runtime.json.

Reply:
Are we still building it in the master branch?

I still think we should create the property to follow the existing pattern.  Even if the property is only ever hard-coded to `2.0.0`

Reply:
Yes, it's building in master but doesn't need to.  There are no changes since 2.0.  I brought it back from deletion to build it just once for 2.0.0.  We can clean it up in corefx-master now.